### PR TITLE
Button Element

### PR DESCRIPTION
**Pull request by bajajcodes**
- Buttons are in seven different options default, full-width, solid, outlined, rounded, icon and floating-action.
- Buttons sizes has three options default, small, and large.
- Buttons colors comes in five options default, primary, seconday, danger and success.
- Buttons on hover change to littel dark color shade of their background color.
- Buttons on disabled true, changes their opacity to half and no-cursor allowed options set to true.(to visualize disabled button)
- Fulll-wdith button takes hundred precent size of parent's width.
- Icon buttons have icon as child inside the button.
- Floating action buttons comes in two options with and without background-color.

127 0 0 1_5500_examples_button html (1)

### CODE REVIEW

File: css/elements/button.css [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    padding: var(--size-0);
+}
+
+.btn-fab-nbg{
+    color: initial;
+    border: var(--size-0);
+}
```
Comment (arp99) [inline css/elements/button.css]: The class Name **.btn-fab** could have been better, Could you explain what styles this class applies?.

**Review by arp99 — COMMENTED** [review]
Looks good, suggested some changes.

**Comment by arp99** [comment]
> * Buttons are in seven different options default, full-width, solid, outlined, rounded, icon and floating-action.
> * Buttons sizes has three options default, small, and large.
> * Buttons colors comes in five options default, primary, seconday, danger and success.
> * Buttons on hover change to littel dark color shade of their background color.
> * Buttons on disabled true, changes their opacity to half and no-cursor allowed options set to true.(to visualize disabled button)
> * Fulll-wdith button takes hundred precent size of parent's width.
> * Icon buttons have icon as child inside the button.
> * Floating action buttons comes in two options with and without background-color.
> 
> 127 0 0 1_5500_examples_button html (1)

Good PR description 😊

**Comment by saadahmadhere** [comment]
looks good to me.

File: css/elements/button.css (hunk 1 above)
Comment (bajajcodes) [reply css/elements/button.css]: The classes **.btn-fab** and **.btn-fab-nbg**reperesents floating action button with background and without background color and I agree for class naming could be better.

I have changed class names to **.btn-floating** and **.btn-floating-wbg**.

Comment (arp99) [reply css/elements/button.css]: Yeah now new class names seems better

**Review by arp99 — APPROVED** [review]
The overall code looks good, suggested changes are done accordingly

**Pull request merged**